cmd/start.go

```diff
@@ -91,3 +117,35 @@ func trapSignal(done func()) {
 	// call the cleanup func
 	done()
 }
+
+// UpdateClientsFromChains takes src, dst chains, threshold time and update clients based on expiry time
+func UpdateClientsFromChains(src, dst *relayer.Chain, thresholdTime time.Duration) (time.Duration, error) {
+	var (
+		srcTimeExpiry, dstTimeExpiry time.Duration
+		err                          error
+	)
+
+	eg := new(errgroup.Group)
+	eg.Go(func() error {
+		srcTimeExpiry, err = relayer.GetClientAndUpdate(src, dst, thresholdTime)
+		return err
+	})
+	eg.Go(func() error {
+		dstTimeExpiry, err = relayer.GetClientAndUpdate(dst, src, thresholdTime)
+		return err
+	})
+	if err := eg.Wait(); err != nil {
+		return 0, err
+	}
+
+	if srcTimeExpiry <= 0 || dstTimeExpiry <= 0 {
+		return 0, fmt.Errorf("one of the clients (%s,%s) trusting period is 0 or expired",
```

Comment:
I think it'd be useful to separate these checks and tell the user which client on which chain is expired. Also I believe it is not possible for the trusting period to be 0